modules/virt-creating-a-service-backed-by-virtual-machines.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+[id="virt-creating-a-service-backed-by-virtual-machines_{context}"]
+
+= Creating a service backed by virtual machines
+
+To create a service backed by a virtual machine, you start the `VirtualMachineInstance`,
+if it is not already running. Once the instance is running, you connect to the instance, and
+create a service object to expose the instance.
+
+The following service types are supported:
+
+* `ClusterIP` - Connect using service IP and service port inside the cluster network.
+* `NodePort` - Connect using node IP and node port outside the cluster network.
+* `LoadBalancer` - Connect using public IP and port.
```

Comment:
You can also connect by name (actually, that's the recommended way); the service exposes the hostname of the VMI to the internal cluster (`ClusterIP` service) or external world (`NodePort`).

Reply:
Changed to:

Connect to the service by specifying the service name. 

The service exposes the host name of the VMI to the internal cluster, 
using the ClusterIP service, or externally, using the NodePort service.

@maiqueb - Do we want to mention type LoadBalancer at all in the downstream doc?

Reply:
@phoracek said: "We can mention [LoadBalancer], but probably just as an option and reference https://docs.openshift.com/container-platform/4.5/networking/configuring_ingress_cluster_traffic/configuring-externalip.html for more info.

To this end, I am including a link to this section in the Additional Resources section.